addons/account/models/account_journal_dashboard.py

```diff
@@ -281,11 +281,6 @@ def get_journal_dashboard_datas(self):
             if read:
                 number_to_check = read[0]['__count']
                 to_check_balance = read[0]['amount_total']
-        elif self.type == 'general':
-            read = self.env['account.move'].read_group([('journal_id', '=', self.id), ('to_check', '=', True)], ['amount_total'], 'journal_id', lazy=False)
```

Comment:
Is it expected that this part is in this PR? Looks unrelated

Reply:
Indeed, bad rebase I guess